client/src/components/FlightDetails.vue

```diff
@@ -207,11 +207,25 @@ import { computed } from "vue";
 import useUser from "@/composables/useUser";
 import useFlight from "@/composables/useFlight";
 import { getbaseURL } from "@/helper/baseUrlHelper";
+import { checkIfDateIsDaysBeforeToday } from "../helper/utils";
 
-const { getUserId } = useUser();
+const { getUserId, hasElevatedRole } = useUser();
 const { flight } = useFlight();
 
-const showEditButton = computed(() => flight.value.userId === getUserId.value);
+const DAYS_FLIGHT_CHANGEABLE = 14;
```

Comment:
Wenn wir es im client checken:

Dafür gibt es in common/constants.js einen Ort. 
`import Constants from "@/common/Constants";
`